### PR TITLE
UI - Support New Auto Approval Record and LTSA Endpoints

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:* https://github.com/bcgov/entity/issues/22631
https://github.com/bcgov/entity/issues/22632

*Description of changes:*
Application detail Auto Approval and LTSA pages are now working.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-109-swlf73p6.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-109-swlf73p6.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-109-swlf73p6.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-109-swlf73p6.web.app

**Comment by shaangill025** [comment]
@rstens I created an application to test, https://strr-ui-dev--pr-109-swlf73p6.web.app/application-details/25 but this hasn't gone through auto approval process yet. Maybe the dev job doesn't run during off hours.

I tested it locally by manually adding data.
```
INSERT INTO public.auto_approval_records(
    id, record, creation_date, application_id)
    VALUES (1, 
    '{
        "renting": true,
        "service_provider": false,
        "pr_exempt": false,
        "address_match": true,
        "business_license_required": true,
        "business_license_required_not_provided": false,
        "business_license_required_provided": true,
        "business_license_not_required_not_provided": false,
        "title_check": true
    }', 
    '2024-08-20 10:00:00', 
    1);


INSERT INTO public.ltsa(
    id, record, creation_date, application_id)
    VALUES (1, 
    '{
        "titleStatus": "ACTIVE",
        "titleIdentifier": {
            "titleNumber": "CA7654321",
            "landTitleDistrict": "VICTORIA"
        },
        "tombstone": {
            "applicationReceivedDate": "2024-07-15",
            "enteredDate": "2024-07-16",
            "titleRemarks": "Test LTSA record",
            "marketValueAmount": "500000",
            "fromTitles": [
                {
                    "titleNumber": "CA7654320",
                    "landTitleDistrict": "VICTORIA"
                }
            ],
            "natureOfTransfers": [
                {
                    "transferReason": "SALE"
                }
            ]
        },
        "taxAuthorities": [
            {
                "authorityName": "City of Victoria"
            }
        ],
        "ownershipGroups": [
            {
                "jointTenancyIndication": true,
                "interestFractionNumerator": "1",
                "interestFractionDenominator": "1",
                "ownershipRemarks": "Joint Tenants",
                "titleOwners": [
                    {
                        "lastNameOrCorpName1": "Smith",
                        "givenName": "John",
                        "incorporationNumber": "",
                        "occupationDescription": "Engineer",
                        "address": {
                            "addressLine1": "123 Main St",
                            "addressLine2": "",
                            "city": "Victoria",
                            "province": "BC",
                            "provinceName": "British Columbia",
                            "country": "Canada",
                            "postalCode": "V8X 1X1"
                        }
                    },
                    {
                        "lastNameOrCorpName1": "Smith",
                        "givenName": "Jane",
                        "incorporationNumber": "",
                        "occupationDescription": "Teacher",
                        "address": {
                            "addressLine1": "123 Main St",
                            "addressLine2": "",
                            "city": "Victoria",
                            "province": "BC",
                            "provinceName": "British Columbia",
                            "country": "Canada",
                            "postalCode": "V8X 1X1"
                        }
                    }
                ]
            }
        ],
        "descriptionsOfLand": [
            {
                "parcelIdentifier": "012-345-678",
                "fullLegalDescription": "LOT A, PLAN VIP12345, SECTION 1, VICTORIA LAND DISTRICT",
                "parcelStatus": "ACTIVE"
            }
        ]
    }', 
    '2024-08-20 10:30:00', 
    1);
```
image
image

File: strr-web/interfaces/auto-approval-data-i.ts [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-    business_license_required_not_provided: boolean | null,
-    business_license_required_provided: boolean | null,
-    business_license_not_required_not_provided: boolean | null,
-    title_check: boolean | null
+    id: number,
+    applicationId: number,
+    creationDate: string,
+    record: {
+      renting?: boolean,
+      service_provider?: boolean,
```
Comment (dimak1) [inline strr-web/interfaces/auto-approval-data-i.ts:8]: Is there a way to make props camelCase for consistency? In WEB and API sides.

**Review by dimak1 — APPROVED** [review]
Looks good, just one comment.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-109-swlf73p6.web.app

**Pull request merged**